app.rb

```diff
@@ -1,8 +1,40 @@
-require 'sinatra/base'
+require 'sinatra'
+require 'sinatra/reloader' if development?
+require './lib/peep_dao'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  # :nocov:
+  configure :development do
+    register Sinatra::Reloader
+  end
+  # :nocov:
+
+  enable :sessions
+
+  get '/' do
+    @button_value = session[:reverse].nil? ? "Reverse" : "Normal"
+    reverse = session[:reverse].nil? ? false : true
+    @peeps = PeepDao.all(reverse, session[:filter])
+    erb(:index)
+  end
```

Comment:
Great use of session throughout your app file. I really like how you've included a reverse button to set the order of the peeps, very clever!